addons/web_editor/static/src/xml/wysiwyg.xml

```diff
@@ -156,7 +155,7 @@
                     </div>
                 </div>
             </div>
-
+            <t t-raw="extraUIElements"/>
```

Comment:
Could use `t-raw="0"` in this case, but fine like that too. Technically ultra-slightly more robust with `t-raw="0"` to me, it would prevent a stupid refactoring to someday allow user content in `extraUIElements`